packages/hydrogen/src/customer/auth.helpers.ts

```diff
@@ -191,7 +191,7 @@ export async function checkExpires({
         throw new BadRequest(
           'Unauthorized',
           'Login before querying the Customer Account API.',
-        );
+        ).headers.set('Set-Cookie', await session.commit());
```

Comment:
Does this work? I think `headers.set(...)` returns `undefined` so this might be equivalent to `throw undefined` 🤔

Reply:
I guess not, the test failure caught it.